packages/react-responsive/package.json

```diff
@@ -9,10 +9,14 @@
   "module": "lib/react-responsive.modern.js",
   "types": "lib/index.d.ts",
   "exports": {
-    "require": "./dist/react-responsive.js",
-    "import": "./dist/react-responsive.modern.js",
-    "browser": "./dist/react-responsive.modern.js",
-    "umd": "./dist/react-responsive.umd.js"
+    ".": {
+      "require": "./lib/react-responsive.js",
+      "import": "./lib/react-responsive.modern.mjs",
+      "browser": "./lib/react-responsive.modern.js",
+      "umd": "./lib/react-responsive.umd.js"
+    },
+    "./package.json": "./package.json",
+    "./": "./"
```

Comment:
Note that folder patterns are deprecated (https://nodejs.org/api/packages.html#packages_subpath_folder_mappings)

Reply:
good to know